.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,22 @@
+name: "Run ShellCheck"
```

Comment:
Would it not be better going with the more popular `ludeeus/action-shellcheck` ?

Reply:
This is the one that popped up first when I was looking for shellcheck actions, and has a higher cadence of development